extensions/amp-form/0.1/amp-form.js

```diff
@@ -20,6 +20,7 @@ import {assertHttpsUrl} from '../../../src/url';
 import {user, rethrowAsync} from '../../../src/log';
 import {onDocumentReady} from '../../../src/document-ready';
 import {xhrFor} from '../../../src/xhr';
+import {getCorsUrl} from '../../../src/service/xhr-impl';
```

Comment:
Do I move this out of `xhr-impl` and into `url.js` maybe to fix the dependencies error travis is running into?